### PR TITLE
PXD-647 Feat/json and BibTeX endpoints for core metadata

### PR DESCRIPTION
**Pull request by paulineribeyre**

### CODE REVIEW

File: pidgin/app.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    #file_type = send_query(query_txt)
+    # reformat output
+
+    # get the list of available types from the graphql schema
+    query_txt = '{ __schema { types { name } } }'
+    all_file_types = send_query(query_txt)
+    all_file_types = get_nested_dict_values(all_file_types.get_json())
+
+    # for each type, check if it contains the file with this object_id
+    for t in all_file_types:
```
Comment (zflamig) [inline pidgin/app.py]: You can actually write a single query that does this and then just check all the results
`{
  aligned_reads_index(object_id:"dg.4503/e6961203-1a04-4136-98dc-b5780d85a3fc") {
    object_id
    core_metadata_collections {
      title description creator contributor coverage
    }
  }
  submitted_aligned_reads(object_id:"dg.4503/e6961203-1a04-4136-98dc-b5780d85a3fc") {
    object_id
    core_metadata_collections {
      title description creator contributor coverage
    }
  }
}`

Comment (paulineribeyre) [reply pidgin/app.py]: This doesn't work because some file types don't have "object_id" so the whole query returns an error

Comment (zflamig) [reply pidgin/app.py]: You can change your schema query to `__schema { types { name fields { name } } }` and then filter the results by types with object_id

**Review by zflamig — APPROVED** [review]

**Comment by paulineribeyre** [comment]
pidgin now uses datanode to get the type of file from the object_id. @zflamig can you review it?

File: run.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from pidgin.app import app
+
+app.config['API_URL'] = 'https://pauline.planx-pla.net/api/v0/submission/graphql/'
```
Comment (zflamig) [inline run.py]: lets change to localhost instead here

Comment (paulineribeyre) [reply run.py]: done

Comment (zflamig) [reply run.py]: Merge it!

**Pull request merged**